scripts/MakeSets.py

```diff
@@ -11,14 +11,14 @@ def main():
     ## MODEL PARAMETERS
     ####################################
 
-    continent = [functions.openYaml().get('continent')]
-    emissions = functions.openYaml().get('emissions')
-    techsMaster = functions.openYaml().get('techs_master')
-    rnwFuels = functions.openYaml().get('rnw_fuels')
-    mineFuels = functions.openYaml().get('mine_fuels')
-    stoTechs = functions.openYaml().get('sto_techs')
+    continentList = [functions.getFromYaml('continent')]
```

Comment:
brackets

Reply:
I think brackets to turn this into a list are necessary either here, or where continentList is called